Marlin/src/module/planner.cpp

```diff
@@ -874,6 +874,10 @@ void Planner::calculate_trapezoid_for_block(block_t * const block, const_float_t
       const float comp = extruder_advance_K[E_INDEX_N(block->extruder)] * block->steps.e / block->step_event_count;
       block->max_adv_steps = cruise_rate * comp;
       block->final_adv_steps = final_rate * comp;
+      #if ENABLED(LA_ZERO_SLOWDOWN)
+        block->la_step_rate = 0;
```

Comment:
Could this be a static variable of the stepper class or the planner to save ram?
I think it could since blocks run one at a time.

Reply:
In principle the LA or acceleration parameters could change between blocks. So I think `la_step_rate` has to be stored for each block.